src/store/index.js

```diff
@@ -263,8 +312,8 @@ export default createStore({
       commit(UPDATE_TRIGGER, { id, updatedTrigger });
       commit(CLOSE_APP_MODAL);
     },
-    showAlert({ commit }, { active, success, message }) {
-      commit(SET_ALERT, { active, success, message });
+    hideAlert({ commit }, { active }) {
```

Comment:
hideAlert will always set the visibility to false, so you don't need a parameter here